xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -229,8 +229,8 @@ enum AppAction: Hashable {
     case completeIndexPeers(results: [PeerIndexResult])
 
     /// Purge the contents of a sphere from the database
-    case purgePeer(_ did: Did)
-    case succeedPurgePeer(_ did: Did)
+    case purgePeer(_ petname: Petname)
+    case succeedPurgePeer(_ petname: Petname)
```

Comment:
Using petname instead of DID allows us to handle multiple names for the same user